src/repositories/__tests__/request-repository.test.ts

```diff
@@ -863,6 +864,24 @@ describe('request repository test', () => {
       const retrieved = await requestRepository.findByCid(unrelatedStreamRequest.cid)
       expect(retrieved).toEqual(unrelatedStreamRequest)
     })
+
+    test('mark regardless of time', async () => {
+      const streamId = randomStreamID()
+      const requestsP = times(3).map(async (n) => {
+        const request = new Request({
+          cid: randomCID().toString(),
+          streamId: streamId.toString(),
+          timestamp: ONE_HOUR_AGO.plus({ minute: n }).toJSDate(),
+          status: RequestStatus.PENDING,
+          origin: 'same-origin',
+        })
+        return requestRepository.createOrUpdate(request)
+      })
+      const requests: Array<Request> = await Promise.all(requestsP)
+      expectPresent(requests[0])
+      const rowsAffected = await requestRepository.markReplaced(requests[0])
+      expect(rowsAffected).toEqual(requests.length - 1) // Mark every request except the last one
```

Comment:
can we check which requests get marked replaced not just the number of requests?  Otherwise it seems like if we did the date comparisons wrong and marked the wrong request as replaced this test would still pass

Reply:
yup